composer.json

```diff
@@ -13,9 +13,11 @@
             "mindplay\\sql\\": "src/"
         }
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
ok, why not use `2.*@dev` for that then? Just to confirm if you add the above lines does it helps people not to add minimum-stability flag on the root of the composer.json on their project?